apps/ui/src/components/EnsConfiguratorOffchain.vue

```diff
@@ -41,6 +41,12 @@ const isTestnet = computed(() => {
   return getNetwork(props.networkId).name.includes('testnet');
 });
 
+const ENS_URL = computed(() =>
+  isTestnet.value
+    ? 'https://sepolia.app.ens.domains'
+    : 'https://app.ens.domains'
+);
+
```

Comment:
```suggestion
const ensUrl = computed(() =>
  isTestnet.value
    ? 'https://sepolia.app.ens.domains'
    : 'https://app.ens.domains'
);

```

As it's computed and not at top of the file I think we can just normal casing.